StreamVideo.xcodeproj/project.pbxproj

```diff
@@ -1679,6 +1682,7 @@
 		842D8BBA2865B31B00801910 = {
 			isa = PBXGroup;
 			children = (
+				829EF8782A9364630045D166 /* PrivacyInfo.xcprivacy */,
```

Comment:
What about the LLC? Shouldn't it be included there as well?

Reply:
It was added to LLC, but I forgot to add it to SwiftUI. Added now